apps/editing-toolkit/editing-toolkit-plugin/wpcom-global-styles/index.php

```diff
@@ -117,10 +117,10 @@ function wpcom_block_global_styles_frontend( $theme_json ) {
 		return $theme_json;
 	}
 
-	if ( class_exists( 'WP_Theme_JSON_Data' ) ) {
-		return new WP_Theme_JSON_Data( array(), 'custom' );
-	} elseif ( class_exists( 'WP_Theme_JSON_Data_Gutenberg' ) ) {
+	if ( class_exists( 'WP_Theme_JSON_Data_Gutenberg' ) ) {
 		return new WP_Theme_JSON_Data_Gutenberg( array(), 'custom' );
+	} elseif ( class_exists( 'WP_Theme_JSON_Data' ) ) {
+		return new WP_Theme_JSON_Data( array(), 'custom' );
```

Comment:
I'm guessing from this change that the order in which we check for the existence of classes is important, maybe we could add a comment explaining why it's important?

Reply:
Yup, good call. The reasoning is that Gutenberg always overrides Core, so we should use their tools when available.